VultisigApp/VultisigApp/Views/Components/Cells/CoinCell.swift

```diff
@@ -16,10 +16,15 @@ struct CoinCell: View {
     }
     
     var logo: some View {
-        ImageView(coin.logo, size: CGSize(width: 32, height: 32))
-            .cornerRadius(50)
+        Group {
+            if coin.logo.hasPrefix("https://") {
+                AsyncImageView(source: .remote(URL(string: coin.logo)), size: CGSize(width: 32, height: 32), ticker: coin.ticker)
```

Comment:
ImageView.swift have init(_ string: String, size: CGSize) that resolves source

Reply:
That was not working. On the main branch, we have this error.

Reply:
Fixed, now we have only one component for images. All encapsulated.